src/keystone/api_only_app/api_only.mak

```diff
@@ -0,0 +1,70 @@
+# git clone keystone
```

Comment:
Is a call to `make -f api_only.mak` something that can be run in CI?

(John was concerned about over-burdening CI builds w/ s/w downloads + setup that might bloat CI run-times.)

I am just bringing this to your attention to evaluate executing these make commands, so that things do not regress in future and [untested] build instructions start to fail.